src/applications/rated-disabilities/components/NeedHelp.jsx

```diff
@@ -7,7 +7,7 @@ export default function NeedHelp() {
       <div slot="content">
         <p>
           You can call us at <va-telephone contact={CONTACTS.VA_BENEFITS} />.
-          We’re We’re here Monday through Friday, 8:00 a.m to 9:00 p.m. ET. If
+          We’re here Monday through Friday, 8:00 a.m to 9:00 p.m. ET. If have
```

Comment:
Should this say `If you have` instead of `If have`?

Reply:
updated !